conans/test/integration/toolchains/android/test_using_meson.py

```diff
@@ -0,0 +1,164 @@
+import os
+import pytest
+import platform
+import textwrap
+import unittest
+
+from parameterized import parameterized
+
+from conans.test.assets.sources import gen_function_cpp, gen_function_h
+from conans.test.utils.tools import TestClient
+
+from ..test_meson import get_meson_version
```

Comment:
```suggestion
from conans.test.integration.toolchains.test_meson import get_meson_version
```

Why not use `Meson.get_version()` static method?